data/resources/style.css

```diff
@@ -263,14 +270,21 @@ messagemedia overlay > messageindicators, messagesticker messageindicators {
 
 /* Placeholder */
 .message-entry > overlay > label {
-  margin: 0 9px;
+  margin-left: 9px;
+}
+
+.message-entry > overlay > entry image {
+  margin: 9px;
 }
 
 .message-entry > overlay > entry {
   padding: 0;
+  margin: 7px 0;
```

Comment:
Why this extra margin?

Reply:
That's without margin

![image](https://user-images.githubusercontent.com/63008755/195670412-3f1c660d-3d90-4d18-995b-915aa8294aa4.png)

Reply:
Weird, I can't reproduce that.

Reply:
By default it have these margins

![image](https://user-images.githubusercontent.com/63008755/195819245-6b4b56ea-36e0-4deb-b690-0cedcdd2d06a.png)

Reply:
And if I understand correctly it has 34px height 
![image](https://user-images.githubusercontent.com/63008755/195820345-ca9a8338-30d2-4b23-8b2a-8cbe3c388bf3.png)

But default message height is 31px (maybe we can increase it to 32px?)

So I've increased margin to reduse height to 32px

Reply:
I need to reduse height by some other way

Reply:
Because if entry and message will have the same height, they will have the same shape. Like on iOS

![image](https://user-images.githubusercontent.com/63008755/195822253-1a436dae-0baf-4d72-ae5f-71ee84c762f5.png)

Reply:
Ok i'll remove this margins, but do you sure that you don't want to use markdown support like in posidon Paper?

![image](https://user-images.githubusercontent.com/63008755/195871129-3239bf9a-78d7-4a8b-9114-466ee9e51485.png)

Reply:
In that case I would like the entry to display \*\***hello****, while the message bubble displays **hello**. So, they would have different chars number.

Reply:
![Screenshot from 2022-10-14 17-51-19](https://user-images.githubusercontent.com/63008755/195876587-19a80125-90d5-4053-a087-2c6943fdc608.png)

i'm not sure that regular telegram user knows markdown, so it will use shortcuts and will not expect those strange symbol

Reply:
The margin is still there. They are not needed because the .toolbar style should already apply it automatically to all the children.